tests/python/frontend/keras/test_forward.py

```diff
@@ -123,6 +123,8 @@ def test_forward_activations():
                  keras.layers.Activation('selu'),
                  keras.layers.ReLU(),
                  keras.layers.ReLU(max_value=6.),
+                 keras.layers.ReLU(max_value=6., threshold=0.),
+                 keras.layers.ReLU(max_value=6., threshold=1.),
```

Comment:
please add test case for _"negative_slope"_ as well, the result is always 0 since the default for this _negative_slope_ is 0.0

Reply:
Thank you for the suggestion, I will add the test case for negative_slope is 0.0